### PR TITLE
[ci][lcnc] add trt-llm container to no code tests

### PR DESCRIPTION
**Pull request by siddvenk**
## Description ##

Adds trt-llm to no code suite.

I've tested portions of the full suite independently, and i'm pretty confident everything will work together.

Struggling with capacity to test the full suite.

This test suite will take a LONG time to run. Couple options:
- add a new input: `run_all` that allows you to run all the tests. The default would be a `$RANDOM` number. This number could be used to run half the tests if even, other half the tests if odd.
- reduce the overall test suite to limit time

### CODE REVIEW

File: .github/workflows/lmi-no-code.yml [hunk 1]
```diff
@@ -186,54 +188,54 @@ jobs:
...
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models ${{ matrix.container }} \
           serve
           python3 llm/client.py no_code baichuan-13b
           docker rm -f $(docker ps -aq)
       - name: Qwen-1.5 lmi container
         working-directory: tests/integration
+        if: ${{ matrix.container == 'deepspeed' }}
         run: |
           rm -rf models
           echo -en "HF_MODEL_ID=Qwen/Qwen1.5-14B" > docker_env
```
Comment (lanking520) [inline .github/workflows/lmi-no-code.yml:227]: qwen 1.5 is qwen2?

File: .github/workflows/lmi-no-code.yml [hunk 2]
```diff
@@ -167,17 +169,17 @@ jobs:
...
           docker rm -f $(docker ps -aq)
       - name: Gemma-7b lmi container
-        if: ${{ matrix.container }} == "deepspeed"
+        if: ${{ matrix.container  == 'deepspeed' }}
         working-directory: tests/integration
         run: |
           rm -rf models
           echo -en "HF_MODEL_ID=s3://djl-llm/gemma-7b/" > docker_env
-          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models ${{ matrix.container }} \
```
Comment (lanking520) [inline .github/workflows/lmi-no-code.yml:182]: remove gemma for trtllm

File: .github/workflows/lmi-no-code.yml (hunk 1 above)
Comment (siddvenk) [reply .github/workflows/lmi-no-code.yml:227]: yea https://huggingface.co/Qwen/Qwen1.5-0.5B/blob/main/config.json

File: .github/workflows/lmi-no-code.yml (hunk 2 above)
Comment (siddvenk) [reply .github/workflows/lmi-no-code.yml:182]: This only runs for deepspeed container: ` if: ${{ matrix.container  == 'deepspeed' }}`

**Review by lanking520 — APPROVED** [review]

**Pull request merged**